Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs

```diff
@@ -289,7 +289,7 @@ public void TestParseTreatMissingFieldsAsNullSetToTrueNoHeaderWithColumnSpecific
             Assert.IsNull(itemArray[6]);

```

Comment:
https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L4
This namespace is not used, so it can be removed.

https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L197
Please use `IsNull` instead of `AreEqual`.

https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L205
Same here.

https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L233
Same here.

https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L241
Same here.